pysph/sph/surface_tension.py

```diff
@@ -444,7 +636,90 @@ def loop(self, d_idx, d_au, d_av, d_aw, d_kappa,
         d_aw[d_idx] += tmp * d_cz[d_idx]
 
 
+class AdamiReproducingDivergence(Equation):
+    r"""Reproducing divergence approximation Eq. (20) in [A10] to
```

Comment:
Just make sure that [A10] is defined.

Reply:
It's defined in the beginning of the codes.